src/algebra/homology/chain_complex.lean

```diff
@@ -113,6 +114,31 @@ local attribute [instance] has_zero_object.has_zero
 instance : inhabited (homological_complex (discrete punit) b) := ⟨0⟩
 end
 
+/-- Map a `homological_complex` with respect to an additive functor. -/
+@[simps]
+def map {C D : Type*} [category C] [category D] [preadditive C] [preadditive D]
+  (Cs : homological_complex C b) (F : C ⥤ D) [functor.additive F] : homological_complex D b :=
```

Comment:
Could we move this to the `functor` namespace, and rename to `map_homological_complex`? This would be parallel with the various `map_iso`, `map_cone`, ... etc we already have.

Reply:
Done!